docs/why.rst

```diff
@@ -0,0 +1,111 @@
+Why µsort?
+==========
+
+µsort was originally designed with safety as the top priority–"first, do no harm".
```

Comment:
I would prefer an em dash with spaces either side.  As-rendered this is an en dash?

Reply:
It was meant to be an em dash, but I frequently miss pressing shift in time and get an en dash instead. 😅  I will fix that.

That said, most style guides recommend *not* having spaces around the em dash. Eg, https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use and https://style.mla.org/is-it-wrong-to-type-spaces-before-and-after-a-dash/